README.md

```diff
@@ -104,6 +104,16 @@ version supported by the given `ansible-core-version` **(DEFAULT: `auto`)**
 Extra command to invoke before ansible-test **(OPTIONAL)**
 
 
+### `pull-request-change-detection`
+
+Whether to use change detection for pull requests. If set to `true`, will
+use change detection to determine changed files against the target branch,
+and will not upload code coverage results. If the invocation is not from a
+pull request, this option is ignored. Note that this requires
+`collection-src-directory` to be empty or something ending with
+`ansible_collections/{namespace}/{name}`. **(DEFAULT: `false`)**
```

Comment:
Why? I'm pretty sure we copy it to the right location: https://github.com/felixfontein/ansible-test-gh-action/blob/change-detection/action.yml#L366.

Reply:
When we copy it, it is no longer part of the git repository, and ansible-test cannot figure out the changed files from the git repository.

Reply:
Ouch... That's unfortunate. But there's a solution — make use `git worktree` instead of copying blindly. This way, `git` commands will have access to the history because worktrees are linked to the main bare Git repo.
@felixfontein do you mind exploring this angle? It should be as straightforward as replacing `cp` with a `git worktree add`.

Reply:
`git worktree add` creates a new checkout of the same repository with the *complete* directory structure for another (or new) branch. I don't see how this should help here.

Reply:
@felixfontein you said that it should be a Git repo. So we could as well mandate that directory to be a Git checkout and just mirror it to the right place, as opposed to shifting the mental overhead of having to remember to keep it a Git repo in the right folder.

Reply:
Do you mean `collection-src-directory`?

Reply:
Yes, this way the users wouldn't need to create a directory structure for it.

Reply:
97de31dd64ab4bee4a0a54d51ba62ea267cccb41.